lib/linguist/languages.yml

```diff
@@ -1457,6 +1457,29 @@ Gentoo Eclass:
   codemirror_mode: shell
   codemirror_mime_type: text/x-sh
   language_id: 128
+Gerber Image:
+  type: data
+  aliases:
+  - rs-274x
```

Comment:
Yeah? Try reading the actual [format spec](https://www.ucamco.com/files/downloads/file/81/the_gerber_file_format_specification.pdf). It doubles as both a drug-trip *and* a lesson in how not to typeset a document. Page 40 pretty much king-punched the designer in me.

I think page 46 sums up the format's insanity best, though...